ice_in

```diff
@@ -21,31 +21,33 @@
   pointer_file = "./rpointer.ice"
   print_global = .true.
   print_points = .false.
+  restart = .true.
```

Comment:
For the first run, is this flag needs to set to be`.false.`? Refer this [issue](https://github.com/payu-org/payu/issues/349#issue-1825453475)

Reply:
I don't think so? But with the CESM NUOPC driver it's not very clear which of the multiple seemingly-equivalent restart flags are are used and which are ignored. Thanks for pointing this out - I'll try to get to the bottom of this today.

Thanks also for pointing out the payu issue - I'd missed that

Reply:
I looked into this and in the CICE NUOPC cap the `start_type` set in `nuopc.runconfig` is used over the `runtype` set in `ice_in`. Basically (see [here](https://github.com/ESCOMP/CICE/blob/34dc66707f6b691b1689bf36689591af3e8df270/cicecore/drivers/nuopc/cmeps/ice_comp_nuopc.F90#L404)):

- If `start_type` is `"continue"` or `"branch"`, then CICE parameters `restart = .true.`, `runtype = "continue"` and `use_restart_time = .true.` are set;
- If `start_type` is `"startup"` then CICE parameter `runtype = "initial"` is set.

So I think everything should work correctly as I've implemented it. But it's probably clearer to remove `runtype = "initial"` from `ice_in`.

Reply:
Thanks @dougiesquire